src/js/pyodide.ts

```diff
@@ -267,6 +267,7 @@ export async function loadPyodide(
     _node_mounts?: string[];
   } = {},
 ): Promise<PyodideInterface> {
+  const t0_init = new Date();
```

Comment:
Note there is also [performance.now](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) which provides higher-resolution timing, if that's desired.